e2e/.env.sample

```diff
@@ -2,3 +2,6 @@
 USER_NAME=??
 # Workbench login password. Alternatively, set "PASSWORD" environment variable in the yarn command.
 PASSWORD=??
+WRITER_USER=puppetcitester9@fake-research-aou.org
```

Comment:
it would be better to make these "[replace with your writer]" or something like that.  Otherwise some lazy person will just reuse these :)

Reply:
yes, I will merge this PR then make a new (smaller) PR to update this and move `interface IPageUrl` to `types.d.ts`.